src/utils/data/admin-data.js

```diff
@@ -1,6 +1,10 @@
 export default (data) => {
   if (data.records && Array.isArray(data.records)) {
```

Comment:
I believe that the one condition that checks if an array is an array will be enough since we don't need to access another property within records.

```suggestion
  if (Array.isArray(data.records)) {
```